__tests__/app.test.js

```diff
@@ -110,3 +108,49 @@ describe("responds with error for a non existent path", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 - /api/articles/:article_id/comments should respond with an array of comments for the given article_id, each of which should have the relevant properties", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(2);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 3,
+          });
+        });
+      });
+  });
+});
```

Comment:
There is a really important missing test here. When given a valid article id but one that has no associated comments we should expect an empty array back as a response. The article id which should have no comments yet is 2 so the endpoint will be /api/articles/2 where we expect an empty array response.